MkDocs/hpu_8.6/prereqs.md

```diff
@@ -25,14 +25,14 @@ o Click `Create an IBM ID` link
 1. From the grid view...
 2. To add the saved view...click on the `Filter` icon, click on `Edit` next to `Query`, and choose `EUDEMO_ST`. Click `Ok` and `Apply`. 
 
-    ![img](/img/hpu_8.6/prereq1.1.png){: style="height:172px;width:200px;margin-left:40px"}
-    ![img](/img/hpu_8.6/prereq1.2.png){: style="height:172px;width:200px;margin-left:40px"}
-    ![img](/img/hpu_8.6/prereq2.1.png){: style="height:172px;width:200px;margin-left:40px"}
+    ![img](/img/hpu_8.6/prereq1.1.png)
+    ![img](/img/hpu_8.6/prereq1.2.png) 
+    ![img](/img/hpu_8.6/prereq2.1.png) 
 
 3. Click the three dots next to the drop down menu and choose `Save as` to save this group of assets. 
 
-    ![img](/img/hpu_8.6/prereq2.2.png){: style="height:172px;width:200px;margin-left:40px"}
+    ![img](/img/hpu_8.6/prereq2.2.png) 
 
 4. This user will now be able to see this view of assets when they login. 
 
-    ![img](/img/hpu_8.6/prereq4.1.png){: style="height:172px;width:200px;margin-left:40px"}
+    ![img](/img/hpu_8.6/prereq4.1.png) 
```

Comment:
You reference these image files prereq4.1.png  and others but don't include them in your pull request.  You are required to make sure that your pull request is functionally complete without files missing or having errors.  Please add the missing image files to this PR.   I would also suggest not having 2 periods in a file name.  It is difficult to read and may cause problems with file type rendering for an image.

Reply:
YOu are also missing your exercises file.  See example below as was done 
<img width="1211" alt="Screenshot 2022-04-06 at 12 14 41" src="https://user-images.githubusercontent.com/60844296/161962767-36a5ef4e-8cf7-4b4d-8b44-82338ce4f942.png">
 
You also don't appear  to have content in your clean and closing md file.   You need the lab to be complete before you push it to the repo for review

Reply:
The clean and closing files have content, albeit not a lot since there isn't a lot of cleanup to do after this lab